tracer/src/Datadog.Trace/LibDatadog/ServiceDiscovery/TracerMemfdHandle.cs

```diff
@@ -0,0 +1,14 @@
+// <copyright file="TracerMemfdHandle.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+using System.Runtime.InteropServices;
+
+namespace Datadog.Trace.LibDatadog.ServiceDiscovery;
+
+[StructLayout(LayoutKind.Sequential)]
+internal struct TracerMemfdHandle
```

Comment:
Do we need this type? I think you can replace the one field of type `TracerMemfdHandle` with `int`, no?